src/pos/client.ts

```diff
@@ -232,17 +232,24 @@ export class PoolProtocolImpl extends AbstractProtocolClient {
 
     private stakingPool: StakingPoolImpl | undefined;
 
+    private lastRebalanceTimestamp: number
```

Comment:
run `yarn prettier` to adjust some code styles.

Reply:
Done